pkgs/tools/graphics/sanjuuni/default.nix

```diff
@@ -4,27 +4,27 @@
 , pkg-config
 , ffmpeg
 , poco
+, ocl-icd
+, opencl-clhpp
+  # Opts
+, withOpenCL ? true
```

Comment:
Why would you want to run the program without hardware acceleration?

Reply:
I've seen other packages have options to disable it. OpenCL is not exactly a light dependency; it's also optional for operation of the program.

Reply:
Marking as resolved, feel free to undo this if you have further discussion.

Reply:
That it fails to build is also a pretty good argument to remove this option as it isn't tested and requires extra complexity.

Reply:
> That's not true. We already depend on ffmpeg which is gigantic compared to the two cl packages. snjuuni has a runtime closure of 582.31 MiB with cl and I couldn't test it without it because it didn't compile but I build ocl-icd and opencl-clhpp which have combined a closure size without glibc of not even 1 MiB. Totally ignorable here.

That's a very fair point. I'll remove the option.